front/src/routes/ApplicantsPage.svelte

```diff
@@ -78,6 +79,33 @@
     }));
     loading = false;
   });
+
+  let selectedCommittee;
+
+  const passesCommitteeFilters = (
```

Comment:
I also would rename this function to something along the lines of `filterApplicantsByCommittee` to directly describe the purpose of the function, because `passesCommitteeFilters` is a bit confusing to me